tests/buyers/views/test_buyers.py

```diff
@@ -2255,7 +2244,7 @@ def test_404_if_framework_is_not_live_or_expired(self):
     def test_get_responses(self):
         brief = mock.Mock()
 
-        with po(buyers, 'get_sorted_responses_for_brief') as m:
+        with mock.patch.object(buyers, 'get_sorted_responses_for_brief', autospec=True) as m:
             result = self.instance.get_responses(brief)
 
         assert result == m.return_value
```

Comment:
Not flake8... But should this assertion be inside the scope of the context m above?